packages/audio-filters-web/src/NoiseCancellation.ts

```diff
@@ -85,7 +86,13 @@ export class NoiseCancellation implements INoiseCancellation {
    * Checks if the noise cancellation is supported on this platform.
    * Make sure you call this method before trying to enable the noise cancellation.
    */
-  isSupported = () => KrispSDK.isSupported();
+  isSupported = () => {
+    if (!KrispSDK.isSupported()) {
+      return false;
```

Comment:
It may be easier to know, that this function should always be awaited?
```suggestion
      return Promise.resolve(false);
```

Reply:
The interface will define it as `boolean | Promise<boolean>` anyway for backward compatibility. `await` can still be used with this return signature